js/main.js

```diff
@@ -80,19 +114,47 @@ function switchViews(view) {
       data.view = $viewElements[i].getAttribute('data-view');
     }
   }
+  if ($ul.childElementCount === 0) {
+    $noItems.className = 'center-text';
+  } else {
+    $noItems.className = 'center-text hidden';
+  }
+
 }
 
 window.addEventListener('DOMContentLoaded', appendDOM);
 var $noItems = document.querySelector('.center-text');
-if ($ul.childElementCount === 0) {
-  $noItems.className = 'center-text hidden';
-} else {
-  $noItems.className = 'center-text';
-}
+
 
 function appendDOM() {
   for (var i = 0; i < data.entries.length; i++) {
     entryDOM(data.entries[i]);
   }
   switchViews(data.view);
+  var $icon = document.querySelectorAll('i');
+  for(var i = 0; i < $icon.length; i++){
+    $icon[i].setAttribute("EntryId", $icon.length - i)
```

Comment:
You should name the attribute `data-entry-id` instead so that it is a proper `data-` custom attribute